Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:8-slim
+
+ENV KARMA_BROWSER PhantomJS
```

Comment:
if we could use headless chrome instead of phantomjs i think it would be better in terms of speed and memory usage, for reference : https://hackernoon.com/benchmark-headless-chrome-vs-phantomjs-e7f44c6956c

Reply:
yep will do that

Reply:
@Lucianolo Do you have an idea about a Docker image for Node with Chrome Headless? Because I searched for it and never found an official one.

Reply:
used `lts-alpine` thanks to @xquangdang